requirements.txt

```diff
@@ -67,7 +68,7 @@ rsa>=4.1 # not directly required, pinned by Snyk to avoid a vulnerability
 aiohttp==3.8.1
 aiosignal==1.2.0
 alembic==1.7.6
-amqp==5.0.9
+amqp==5.1.0
```

Comment:
These were updated by `make freeze-requirements`.  Is this expected to have unrelated modules updated when adding a new module to `requirements-app.txt`?

Reply:
Yeah it's expected. We need to review these and be careful. Patch versions are great to have, we should be cautious with minors and I tend to avoid major for now, unless the current PR is dedicated to a dependency bump and we will properly test the application.

Reply:
Makes sense.  To be on the safe side, I've reverted the two minor bumps and left the patch version.